src/components/Feelings/Feelings.module.css

```diff
@@ -39,6 +39,20 @@
   cursor: pointer;
   transition: 0.5s all;
 }
+.feeling .feel_active .active{
```

Comment:
I think this shouldn't be like this. I will help you with the CSS. Please give me few moments and I will tell you the exact CSS code that should be implemented.

Reply:
Yea please I am confused about it